src/Application.php

```diff
@@ -329,6 +329,11 @@ public function initProviders()
         $this->register(new Silex\Provider\SwiftmailerServiceProvider());
         if ($this['config']->get('general/mailoptions')) {
             $this['swiftmailer.options'] = $this['config']->get('general/mailoptions');
+        } else {
+            $this['swiftmailer.use_spool'] = false;
```

Comment:
The transport won't be used if use_spool is set to true:
https://github.com/silexphp/Silex-Providers/blob/master/SwiftmailerServiceProvider.php#L37